frb_example/dart_build_rs/pubspec.yaml

```diff
@@ -3,7 +3,7 @@ description: flutter rust bridge example
 version: 1.0.0
 publish_to: none
 environment:
-  sdk: '>=3.2.0 <4.0.0'
+  sdk: '>=3.3.0 <4.0.0'
```

Comment:
btw you may need to change ci.yaml's flutter/dart version in order to make ci happy

Reply:
Thanks for the tip, done.